server/lib/transferwise.ts

```diff
@@ -0,0 +1,124 @@
+import axios from 'axios';
+import config from 'config';
+import { get, omitBy, isNull } from 'lodash/fp';
```

Comment:
I'm still not 100% bringing these `fp` patterns in the codebase.

Reply:
Thanks for catching that, I also don't think it is a good idea to mix normal lodash and fp.